kafka/protocol/__init__.py

```diff
@@ -6,3 +6,40 @@
     CODEC_NONE, CODEC_GZIP, CODEC_SNAPPY, ALL_CODECS,
     ATTRIBUTE_CODEC_MASK, KafkaProtocol,
 )
+
+API_KEYS = {
```

Comment:
TBH, I created it primarily just to document the values. But there is some potential to switch the values from strings to the versioned list of class structs we have and use it for decoding api messages. That's obviously not in this PR though.